components/layout/Nav.js

```diff
@@ -4,22 +4,25 @@ import useUser from '../auth/User';
 import LogOut from '../auth/LogOut';
 import { useCart } from '../../context/cartState';
 import CartTally from '../cart/CartTally';
+import AccountDropdown from '../elements/AccountDropdown';
 
 const Nav = () => {
   const cart = useCart();
   const user = useUser();
 
   return (
     <StyledNav>
-      <Link href="/products">All Tutorials</Link>
       {user ? (
-        <>
-          <Link href="/my-tutorials">my tutorials</Link>
-          <Link href="/account">account</Link>
-          <LogOut />
+        <section>
+          <AccountDropdown>
```

Comment:
As the `StyledNav` is just the container for multiple lists of links, I would say that this `AccountDropdown` is a  `nav`

Reply:
AGREEEEEEEEEEE